src/sections/downloads-v3/linux-gentoo.marko

```diff
@@ -2,20 +2,11 @@
     <maintained-by><a href="https://github.com/xartin/gentoo-overlay">xartin</a></maintained-by>
```

Comment:
Since this is moving from the overlay I think this can be updated to reference both you and xartin and point to the appropriate repo.

Reply:
I'll change it. The site needs to be generated anyway.